.github/workflows/generate-openapidocs-pr-workflow.yml

```
@@ -2,7 +2,7 @@
 # Licensed under the MIT License.
 # This is a basic workflow to help you get started with Actions
 
-name: WeeklyOpenApiDocsDownload
+name: "Generate Weekly OpenAPIDocs Download PR"
```

Let's use "create" instead of "generate" here since are opening a PR for review. The file name should also be renamed to `create-openapidocs-pr-workflow.yml`
```suggestion
name: "Create Weekly OpenAPIDocs Download PR"
```